packages/components/pie-card-container/src/index.ts

```diff
@@ -31,30 +38,51 @@ export class PieCardContainer extends LitElement implements CardContainerProps {
     @property({ type: Object })
     public aria!: AriaProps;
 
+    /**
+     * Renders the card as an anchor element.
+     *
+     * @private
+     */
+    private renderAnchor (): TemplateResult {
```

Comment:
Sorry i was off on friday so might have missed this PR. I was thinking could we maybe having this function return a template based on the interaction type or "tag" instead? The markup looks quite similar to the button one. Something like `renderCardTemplate` this can be a generic method that handles the interaction type or tag setting all the props then inside the main render we can call either or based on user input.